speaker_verification/sql_utils.py

```diff
@@ -130,14 +141,30 @@ def insert_db_row(table, id, mfcc):
     ----------
     table : str
         Name of table to insert record within.
-    id : str
+    id : int
         Id key for required record within table for insertion.
     mfcc : numpy.array
         MFCC dataset to be inserted within database records.
     """
+    length = len(str(id))
+    if length == 9:
```

Comment:
You should define a constant variable for the [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)). For example: `MAX_ID_LENGTH = 9`

Reply:
I'll remove the changes made to other files besides denoise.py since its not part of this task